monitoring/prometheus/prometheus_staging.yml

```diff
@@ -0,0 +1,15 @@
+global:
+  scrape_interval: 30s
+  external_labels:
+    monitor: 'evalai-monitor'
+
+scrape_configs:
+  - job_name: 'prometheus' 
+    static_configs: 
+      - targets: ['localhost:9090']
+
+  - job_name: 'django-prometheus'
+    metrics_path: '/prometheus/metrics'
+    static_configs:
+      - targets: ['https://staging.eval.ai:8000']
```

Comment:
@burnerlee this won't work. Our django server is behind a nginx which handles redirection. See the nginx config [here](https://github.com/Cloud-CV/EvalAI/blob/master/docker/prod/nodejs/nginx_staging.conf). You can't access django server using port on`https` endpoint

Reply:
I'll take a while.. I have no experience in reading this type of file.. just giving it a read